cmd/root/root.go

```diff
@@ -97,7 +98,28 @@ func flagErrorFunc(c *cobra.Command, err error) error {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute(ctx context.Context, cmd *cobra.Command) error {
-	// TODO: deferred panic recovery
+	defer func() {
+		r := recover()
+
+		// No panic. Return normally.
+		if r == nil {
+			return
+		}
+
+		version := build.GetInfo().Version
+		trace := debug.Stack()
+
+		log.Errorf(ctx, `The Databricks CLI unexpectedly panicked. This should not happen.
```

Comment:
Unexpected implies that it shouldn't happen. The second sentence can be omitted.

Are you sure the logger can be used here? I recommend writing directly to `cmd.OutOrErr()`.

Reply:
Both done.